src/output/plugins/httpd/HttpdClient.cxx

```diff
@@ -169,7 +200,8 @@ HttpdClient::HttpdClient(HttpdOutput &_httpd, UniqueSocketDescriptor _fd,
 			 bool _metadata_supported)
 	:BufferedSocket(_fd.Release(), _loop),
 	 httpd(_httpd),
-	 metadata_supported(_metadata_supported)
+	 metadata_supported(_metadata_supported),
+	 provided_password{""}
```

Comment:
What's the point of this initializer?

Reply:
I don't think it's needed after the last simplification, sorry I missed it.